torch/_inductor/ir.py

```diff
@@ -7202,23 +7205,27 @@ def create(
         cls,
         cond_fn: Subgraph,
         body_fn: Subgraph,
-        operands: List[TensorBox],
+        carried_inputs: List[TensorBox],
+        additional_inputs: List[TensorBox],
     ):
-        operands = [cls.realize_input(x) for x in operands]
+        carried_inputs = [cls.realize_input(x) for x in carried_inputs]
+        additional_inputs = [cls.realize_input(x) for x in additional_inputs]
 
-        fx_operands = V.graph.current_node.args[-1]
-        fake_operands = [x.meta["val"] for x in fx_operands]  # type: ignore[union-attr]
+        fx_carried_inputs = V.graph.current_node.args[-2]
+        fx_additional_inputs = V.graph.current_node.args[-1]
+        fake_carried_inputs = [x.meta["val"] for x in fx_carried_inputs]  # type: ignore[union-attr]
+        fake_additional_inputs = [x.meta["val"] for x in fx_additional_inputs]  # type: ignore[union-attr]
```

Comment:
As these are used identically below for carried and additional inputs, would it make sense to combine them upstream to something like `all_inputs` and then do fx, fake, and use that single list below?